tasks/functions/functions.js

```diff
@@ -21,35 +23,71 @@ function sum(a, b) {
  * }
  */
 function getFullName(object) {
-
+ return object.firstName + " " +  object.lastName ;
 }
-
+const fullName = getFullName ({
+	    firstName: "John",
+	    lastName: "Dou",
+	 });
+console.log(fullName);
 /**
  * write fuction that checks if number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+ if(n % 2 == 0) {
+		console.log("false");
+	}
+	else {
+		console.log("true");
+	}
 }
+ const answer = isOdd(2);
+console.log(answer);
+
 
 /**
  * write function that returns shortest of the words in the given array
  * @example
  * console.log(getShortest(["one", "two", "three"])) // one
  */
+
 function getShortest(wordArray) {
+let shortest = '';
+    if(wordArray.length > 0){
+         for(let i = 0; i < wordArray.length; i++){
+           if(typeof wordArray[i] === 'string')
+             {
+               if(shortest.length == 0) {
+                   shortest = wordArray[i]; continue;
+               } 
+               if(wordArray[i].length < shortest.length){
+                   shortest = wordArray[i]; 
+               }
+            }
+          }
+         }
+          return shortest; 
+       }
+       
+       const output = getShortest(["one", "two", "three"]);
+console.log(output);
+
 
-}
 
 /**
  * write function that returns word google with given numbers of "o" symbols
  * @example
  * console.log(getGoogle(5)) // gooooogle
  */
-function getGoogle(n) {
 
+ function getGoogle(n) { 
+    let str0 = "g"; 
+    let str1 = str0.padEnd((n+1),"o") + "gle"; 
```

Comment:
please try doing it using .repeat() method: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/repeat